controller/persistence/base_store.go

```diff
@@ -108,6 +108,48 @@ func (store *baseStore) baseLoadOneById(tx *bbolt.Tx, id string, entity boltz.Ba
 	return nil
 }
 
+func (store *baseStore) updateEntityNameReferences(bucket *boltz.TypedBucket, rolesSymbol boltz.EntitySetSymbol, entity NamedEdgeEntity, oldName string) {
+	// If the entity name is the same as entity ID, bail out. We don't want to remove any references by ID, since
+	// those take precedence over named references
+	if store.IsEntityPresent(bucket.Tx(), oldName) {
+		return
```

Comment:
i feel like this needs a log, or to return an err maybe? If not - why not?

Reply:
I'll add a log statement